chroma-manager/tests/feature/cli/features/steps/standard_cli.py

```diff
@@ -91,5 +91,5 @@ def step(context, message):
         ok_(message not in "".join(context.stdout.readlines()))
     except AssertionError:
         context.stdout.seek(0)
-        print context.stdout.readlines()
+        print(context.stdout.readlines())
```

Comment:
That should not be necessary.  I think.  GH uses a merge strategy when submitting a patch so any duplicate patches across multiple landings should merge successfully.